<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoinTossing</class>
 <widget class="QDialog" name="CoinTossing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>740</y>
     <width>951</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>7</number>
    </property>
    <item>
     <widget class="QPushButton" name="PB_help_CT">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>American Signs</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: 1px solid black;
background : rgb(186, 184, 175);
border-radius: 10px;
}

QPushButton::hover{
background : rgb(216, 214, 203);
}</string>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="PB_back_home_CT">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>American Signs</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background : rgb(198, 0, 0);
color: white;
border:1px solid blaxk;
border-radius : 10px;
}

QPushButton::hover{
background : rgb(230, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Back Home</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>PB_help_CT</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>PB_back_home_CT</zorder>
  </widget>
  <widget class="QWidget" name="W_status_bar_CT" native="true">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>790</y>
     <width>1031</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(245, 185, 125);</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>American Signs</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Cookies balance</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="L_cookies_balance_CT">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="L_PX_stack_right_CT">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>310</y>
     <width>200</width>
     <height>510</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/stack-of-coins-right.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="L_PX_stack_left_CT">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>310</y>
     <width>201</width>
     <height>510</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/stack-of-coins.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QWidget" name="W_command_panel" native="true">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>380</y>
     <width>591</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#W_command_panel{
	background:rgb(0, 103, 0);
	border: 2px solid  rgb(245, 185, 125);
	border-radius: 20px;
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="PB_start_random_CT">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background :  rgb(246, 170, 80);
border:1px solid blaxk;
border-radius : 20px;
}

QPushButton::hover
{
background  : rgb(253, 253, 0);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/images/coin-flip-pb.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="L_enter_bet_CT">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Javatages</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter your bet : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_enter_bet_CT">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="L_max_bet_CT">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Javatages</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>(Max ..)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_save_bet_CT">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>American Signs</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border : 1px solid black;
border-radius : 10px;
background : rgb(170, 0, 0);
color: white;
}

QPushButton::hover{
background : rgb(148, 0, 0)
}</string>
        </property>
        <property name="text">
         <string>Save my bet</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="L_choose_side_CT">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Javatages</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Choose one side : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="SP_choose_side_CT">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>67</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="L_legend_CT">
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>175</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Javatages</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1 : Heads     2 : Tails</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="L_PX_coin_2_CT">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/coin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="L_PX_coin_3_CT">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/coin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="L_PX_speed_3_CT">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/left-align.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="L_PX_coin_1_CT">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/coin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="L_PX_speed_1_CT">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>10</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/left-align.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="L_PX_coin_4_CT">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>750</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/coin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="L_PX_speed_4_CT">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>684</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/center-align.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="L_PX_speed_2_CT">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/center-align.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="L_PX_flip_coin_CT">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/coin-tails.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="L_PX_speed_5_CT">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>754</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/left-align.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>353</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>American Signs</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Command panel</string>
   </property>
  </widget>
  <zorder>L_PX_speed_5_CT</zorder>
  <zorder>L_PX_coin_4_CT</zorder>
  <zorder>L_PX_coin_3_CT</zorder>
  <zorder>L_PX_stack_left_CT</zorder>
  <zorder>L_PX_stack_right_CT</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>W_status_bar_CT</zorder>
  <zorder>groupBox</zorder>
  <zorder>W_command_panel</zorder>
  <zorder>L_PX_coin_2_CT</zorder>
  <zorder>L_PX_speed_3_CT</zorder>
  <zorder>L_PX_coin_1_CT</zorder>
  <zorder>L_PX_speed_1_CT</zorder>
  <zorder>L_PX_speed_4_CT</zorder>
  <zorder>L_PX_speed_2_CT</zorder>
  <zorder>L_PX_flip_coin_CT</zorder>
  <zorder>label</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
